src/routes/arns.ts

```diff
@@ -66,11 +66,15 @@ arnsRouter.get('/ar-io/resolver/:name', async (req, res) => {
   );
   res.header(headerNames.arnsProcessId, processId);
   res.header(headerNames.arnsResolvedAt, resolvedAt.toString());
+  res.header(headerNames.arnsIndex, index.toString());
+  res.header(headerNames.arnsLimit, limit.toString());
```

Comment:
note: no 402 enforcement here - this should help with debugging

Reply:
@dtfiedler Good idea, but we'll need to update the legacy gateway to enforce the limit too.